server/utils/slugify.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-useless-escape */
+const slugifyTitle = string => {
+  const a = 'àáäâãåăæçèéëêǵḧìíïîḿńǹñòóöôœṕŕßśșțùúüûǘẃẍÿź·/_,:;';
```

Comment:
In this house we do not host single-letter variable names (except _maybe_ in for loops 🙂)

You should still be able to comfortably explain this code if we come back to it at the end of Simulations

Reply:
I have updated this function.